pyppeteer/chromium_downloader.py

```diff
@@ -76,7 +77,14 @@ def download_zip(url: str) -> BytesIO:
     # see https://urllib3.readthedocs.io/en/latest/advanced-usage.html for more
     urllib3.disable_warnings()
 
-    with urllib3.PoolManager() as http:
+    if HTTP_PROXY != '':
+        logger.info('Not using a proxy...')
+        conn = urllib3.PoolManager()
+    else:
+        logger.info('Using proxy: %s' % HTTP_PROXY)
```

Comment:
The else branch will only be executed when HTTP_PROXY is the empty string. I presume you want to invert the condition. Did you test this change?